main.yml

```diff
@@ -46,6 +47,11 @@ insights:
   deployment_repo: https://github.com/RedHatInsights/insights-advisor-frontend-build
   frontend_paths:
   - /insights
+  frontend_exclude:
```

Comment:
Why do we need this?
I noticed you adding this to the manual configs, but I dont think its really necessary.
Shouldnt the "last match wins" in Akamai prevent the need for this?

Reply:
It's "last match wins", but multiple matches can occur, which has caused us issues before. That's why we changed our rules for insights to match `/insights/*` but exclude `/insights/inventory/*`, `/insights/remediations/*`, etc.
We originally decided to do it this way in order to be extra explicit with our app rules; we might be able to avoid using exclusions by forcing the rules to be created in a certain order, though it would be less explicit.

Reply:
Come to think of it, I might be able to generate these exclusions just based on the paths of the other apps in the YAML. If you want, I can take a stab at that once I'm done with my other work today.